Makefile

```diff
@@ -0,0 +1,12 @@
+.phony: all clean test
```

Comment:
Makefile file should not be committed right?

Reply:
This one is ok, it's just for convenience, and won't be overwritten since now CMake works inside `build/`